cogs/wordfilter/wordfilter.py

```diff
@@ -313,13 +355,11 @@ async def _whitelistList(self, ctx):
                 display.append("`{}`".format(channel))
 
             page = paginator.Pages(ctx=ctx, entries=display, show_entry_count=True)
-            page.embed.title = "Whitelisted channels for: **{}**".format(guildName)
+            page.embed.title = "Allowlist channels for: **{}**".format(guildName)

```

Comment:
Shouldn't this use the ssame format that you've used above with the 
`f"Allowlist channels for: {guildName}"` instead of using .format()?

Reply:
Thanks for pointing it out; missed this one. Will fix in next commit.